Day0/Reetwiz/assignment1/render_ppm.c

```diff
@@ -0,0 +1,155 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+typedef struct
+{
+    unsigned int cx, cy, r;
+} CIRCLE;
+
+typedef struct
+{
+    unsigned int cx, cy, width, height;
+} RECT;
+
+typedef struct
+{
+    unsigned char R, G, B;
+} PIX;
+
+typedef struct
+{
+    unsigned int row, column;
+    PIX **pixel; //pixels stored in 2d array of rows and columns
```

Comment:
It's possible to use 1D array, which is preferred for cache, and it's also easier to allocate, use and free